crowbar_framework/config/initializers/delayed_job_config.rb

```diff
@@ -14,6 +14,18 @@
 # limitations under the License.
 #
 
+# Enable proper traceback logging of delayed jobs
+module Delayed
+  class Worker
+    def handle_failed_job_with_logging(job, error)
+      handle_failed_job_without_logging(job, error)
+      Delayed::Worker.logger.error(error.message)
+      Delayed::Worker.logger.error(error.backtrace.join("\n"))
```

Comment:
In crowbar we use always something like: `Delayed::Worker.logger.error("Error processing job: #{error.message} #{error.backtrace.join("\n")}")`

Do you think it makes sense do to the same here?